README.md

```diff
@@ -41,6 +41,8 @@ Note that Nucleus does not add multiplayer or co-op to single player games, the
 - Microsoft Visual C++ 2015-2019 Redistributable (both x86 and x64)
 
 # How can you contribute?
+If you would like to learn all about contributing to this project, whether bug reporting or working on the codebase, check out the [Nucleus Coop Contributing Guide](CONTRIBUTING.md)!
```

Comment:
Thanks for the feedback! 

I hadn't originally planned on tackling the Code of Conduct to try and limit scope, but... that said, I could definitely move the more "behavioral"/soft stuff to a code of conduct doc, if that'd be a helpful thing to add (though not sure if I should do that as a separate PR).

If I did do that, is there anything specific that you think should be moved to that doc / anything else that you think a code of conduct doc would need (other than the more traditional "don't be a jerk" type of stuff)? My initial thought is to move the whole "Communication" section there and then patch in the gaps for an initial Code of Conduct.

In terms of technical dependencies, were you thinking stuff like the version of .net that needs to be installed + a brief bit on what each of the submodule dependencies does? Or something a bit more detailed on them? 

As I think about it, if the guide is starting to get a bit long too as it is, and it'd be good to keep it briefer, do you think it'd make sense to move relevant stuff to a Code of Conduct + maybe link to a more detailed article(s) about the dependencies and create that as a seperate issue to tackle? And then edit in a link to it from the contributor guide when that one is done? Depending on how much detail is needed, I could see that ballooning things quite a bit. 

I wonder if a link to the detailed history doc from the SplitScreen.Me site might be helpful in here too? Maybe as a "read this first to know the development history of what you will be working on" kinda thing?

Thanks again! This is all really good stuff! Like I didn't even know there *was* a development server to work with, so that's exactly the kind of content gap that I was looking to identify.